src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -336,6 +340,14 @@ private static void ItemEqual(string[] actualList, List<FileData> expectedList,
                                         entryStream.Write(new ReadOnlySpan<byte>(buffer, 0, bytesRead));
                                     }
                                 }
+                                else if (writeInChunks)
```

Comment:
can you explain this writeInChunks option that only tests are using..

Reply:
if it's to simulate an innstallStream that only returns <= 5 chars each read, which exercises the CRC calculation, could you pass in such a stream in the test instead?

Reply:
Sure! This was added to hit the failure case in https://github.com/dotnet/corefx/issues/25803 where we were only testing one version of write: the one without an offset. So we didn't have any test coverage where Write was called with a nonzero offset which led to us missing a bug.

Note that this is in the `ZipTestHelper` file.

Reply:
oh disregard, I see this is a test method. Still could use a comment.

Reply:
> Still could use a comment.

Sure thing :)

Reply:
I see it exposes the bug you fixed.